### DISCUSSION TITLE
RU Split 7.2x without proprietary O-RU

### DISCUSSION THREAD

**Discussion by facundopedre** [srsran/srsRAN_Project#927] [General Help]
Hi there

Our research team has been working with srsRAN for the past year, implementing as many split variations as we can. We are now trying to implement the 7.2x split, but currently have no access to any of the O-RUs recommended in the documentation. Is there any way we could implement a RU with PC software? For example, using an SDR (USRP B200) + RPi 4 to emulate the Low-PHY processing layer. Is there any solution (yours or someone else's) available? Or the only way available is implementing DU on the RPi4 and using the SDR as RF? Our final goal would be having CU - DU - RU, all running in separate machines.

We saw another [discussion](https://github.com/srsran/srsRAN_Project/discussions/517) were someone created a RU simulator, but we need to have it working like in this [link](https://docs.srsran.com/projects/project/en/latest/tutorials/source/oranRU/source/index.html#oran-ru-tutorial), using a COTS UE, not just for testing. 

Thank you in advance

**Comment by khooi8913** [comment]
Hi, in case you are still looking for such an implementation, you can check out our project at: https://github.com/NUS-CIR/ProtO-RU. It extends the srsRAN RU emulator and allows a setup exactly as you need, but an RPi4 may not work -- you will require a machine that can be equipped with at least a 10GbE NIC.
